### DISCUSSION TITLE
How to run ls-dyna k files ?

### DISCUSSION THREAD

**Discussion by onion-xiao** [OpenRadioss/OpenRadioss#532] [Q&A] [answered]
I want to know how to run ls-dyna k file with openradioss? How to write the script?

**Comment by roguic** [comment, marked as answer]
Hi Onion-xiao,

If you want to run a ld-dyna f file model, you just need to replace the *.rad file by *.k file in your command line. For example,
./starter_linux64_gf -np 4 -i my_openradioss_model.rad will run openradioss with 4 restart files and will use my_openradioss_model.rad as input model
./starter_linux64_gf -np 3 -i my_other_model.k will run openradioss with 4 restart files and will use my_other_model.k as input model

Best,
Romain

**Reply by onion-xiao** [reply]
Thank you, I've tried and it works!

**Comment by MarianBulla** [comment]
Hi @onion-xiao, there is nothing to do from your side. Just input the ls-dyna k-file as your would do with native Radioss file.
The internal reader will take the file.
Therefore the config files are used.
Sometimes, there is a need of finetuning the ls-dyna model. 
Anyway, you will get an output message file on how it went.
There are also some examples on the wegsite: www.openradioss.org

Best,
Marian

**Comment by imdatozdagci** [comment]
I'm running on Win64 and can run for .rad files without a problem. When I open my run_mpi and .k files in the same file in the terminal, I get the following error. What should I do for solution, win64 doesn't support running directly?
![10](https://github.com/OpenRadioss/OpenRadioss/assets/112629815/a77ec3bb-b4a9-433d-874c-2fe6f47640a2)

**Reply by izy-glitch** [reply]
Yes please help!!!

**Reply by PaulAltair** [reply]
the script you are using is expecting _0000.rad as input, you can run directly from command line (without the script) or modify the script, or use the guis that are available that make submission a little easier: https://github.com/orgs/OpenRadioss/discussions/1231#discussioncomment-6490163

**Reply by li2qn** [reply]
I had the exactly the same problem with MPI, May I ask have you fixed it and how? Thanks for your help.

Best
Lee

**Reply by FGarelli** [reply]
You could try to run directly from command line. Here a base of the code that you could use:

cd <directory of the file>

set OPENRADIOSS_PATH=C:\02_open_radioss\OpenRadioss
set RAD_CFG_PATH=%OPENRADIOSS_PATH%\hm_cfg_files
set RAD_H3D_PATH=%OPENRADIOSS_PATH%\extlib\h3d\lib\win64
set KMP_STACKSIZE=400m
set PATH=%OPENRADIOSS_PATH%\extlib\hm_reader\win64;%PATH%
set PATH=%OPENRADIOSS_PATH%\exec;%PATH%
set PATH=C:\Program Files (x86)\Intel\oneAPI\compiler\2023.1.0\windows\redist\intel64_win\compiler;%PATH% 
call "C:\Program Files (x86)\Intel\oneAPI\setvars.bat" intel64 vs2022
set OMP_NUM_THREADS=1

C:\02_open_radioss\OpenRadioss\exec\starter_win64_db.exe -i <name of the file>.k -np 10 -notrap
mpiexec -delegate -np 10 C:\02_open_radioss\OpenRadioss\exec\engine_win64_impi.exe -i <name of the file>_0001.rad
